wled00/bus_manager.h

```diff
@@ -280,10 +296,12 @@ class BusPwm : public Bus {
 
   //does no index check
   uint32_t getPixelColor(uint16_t pix) {
+    if (!_valid) return 0;
     return ((_data[3] << 24) | (_data[0] << 16) | (_data[1] << 8) | (_data[2]));
   }
 
   void show() {
+    if (!_valid) return;
```

Comment:
A possible workaround would be using a bus type with bit 5 (digital bus) set for virtual, e.g 80-82+ (digging into one extra reserved bit). This will create an invalid digital bus in earlier versions, which has logic that is just barely adequate to protect against disaster in this type of invalid configuration in my testing.

Reply:
BusPwm has a check if IS_PWM() and does not initialize on false. Unfortunately some other functions lack the check for _valid, so we do have an issue. Looking at it, we should add check for _valid to all functions.
Unfortunately even if you bump bus type to 80+ you'll still end up with the problem that show() will call uninitialized PolyBus pointer. Not sure if this is catastrophic but it is there.

Reply:
It fortunately causes no issues, tested it. The reason is that _iType will be I_NONE and this will never call anything on nullptr, but just break out of the switch in bus_wrapper.h.

Would you be fine with me changing bus type to 80?